content/en/docs/advanced-tasks/adding-an-admission-webhook.md

```diff
@@ -0,0 +1,74 @@
+---
+title: "Shipping an operator that includes Admission Webhooks"
+linkTitle: "Admission Webhooks"
+weight: 3
+---
+
+## Defining your Webhook in the ClusterServiceVersion
+
+### Creating an Admission Webhook
+
+OLM is capable of managing the lifecycle of [validating](https://kubernetes.io/docs/reference/access-authn-authz/admission-controllers/#validatingadmissionwebhook) and [mutating](https://kubernetes.io/docs/reference/access-authn-authz/admission-controllers/#mutatingadmissionwebhook) admission webhooks that are shipped alongside your operator. When developing an [admission webhook](/docs/reference/admission-webhooks) that will be managed by OLM you should consider the following constraints.
+
+#### Certificate Authority Constraints
+
+OLM is currently configured to provide each deployment with a single Certificate Authority (CA). The logic that generates and mounts the CA into the deployment was originally used by the [API Service](https://kubernetes.io/docs/tasks/access-kubernetes-api/setup-extension-api-server/) lifecycle logic. As a result:
+
+- The tls Cert file will be mounted to the deployment at `/apiserver.local.config/certificates/apiserver.crt`
+- The tls Key file will be mounted to the deployment at `/apiserver.local.config/certificates/apiserver.key`
+
+#### Admission Webhook Rules Constraints
+
+Additionally, in an attempt to prevent operator from configuring the cluster into an unrecoverable state, OLM does not allow the Rules defined in the WebhookDescription to:
```

Comment:
Devil's Advocate: I don't know what `WebhookDescription` is at this point.

Reply:
Also, I think we want a little more detail on what we mean by "does not allow". Does the CSV get rejected on create/update? is there a failed status? etc?

Reply:
That's fair, will update the doc!